open_instruct/vllm_utils2.py

```diff
@@ -138,9 +138,15 @@ def __init__(self, *args, bundle_indices: list = None, **kwargs):
             os.environ["VLLM_RAY_BUNDLE_INDICES"] = ",".join(map(str, bundle_indices))
             print(f"creating LLM with bundle_indices={bundle_indices}")
 
-        from vllm import LLM
+        if tool_use:
+            # use tool vllm
+            from open_instruct.tool_utils.tool_vllm import ToolUseLLM
```

Comment:
Nice change! Did you end up merging `vllm_utils2.py` and `vllm_utils3.py`?

Reply:
Ah, I didn't, I should...